plugin/storage/memory/memory.go

```diff
@@ -27,18 +27,25 @@ import (
 	"github.com/jaegertracing/jaeger/model"
 	"github.com/jaegertracing/jaeger/model/adjuster"
 	"github.com/jaegertracing/jaeger/pkg/memory/config"
+	"github.com/jaegertracing/jaeger/pkg/tenancy"
 	"github.com/jaegertracing/jaeger/storage/spanstore"
 )
 
 // Store is an in-memory store of traces
 type Store struct {
+	sync.RWMutex
+	config    config.Configuration
```

Comment:
I would recommend keeping config at the Tenant struct level (copy the same one for now), because it seems very reasonable to want per-tenant quota settings rather than one global setting. Or maybe not?

Reply:
Sorry, what I meant was this:
```go
type Store struct {
  // Each tenant gets a copy of default config.
  // In the future this can be extended to contain per-tenant configuration.
  defaultConfig config.Config
}
type Tenant struct {
  config config.Config
}

func (s *Store) newTenant() *Tenant {
  return &Tenant{
    config: s.defaultConfig
  }
}
```